recipes/RunCapabilityPatternNode/src/Application.purs

```diff
@@ -36,11 +36,11 @@ program = do
 derive instance loggerFunctor :: Functor LoggerF
 derive instance getUserNameFunctor :: Functor GetUserNameF
 
-type LOGGER = FProxy LoggerF
-type GET_USER_NAME = FProxy GetUserNameF
+type LOGGER r = (logger :: LoggerF | r)
+type GET_USER_NAME r = (getUserName :: GetUserNameF | r)
```

Comment:
This compiles and works, but I was just guessing my way through this refactor while referencing the [`Run` docs](https://github.com/natefaubion/purescript-run#writing-interpreters).